### DISCUSSION TITLE
wiki grampslet

### DISCUSSION THREAD

**Discussion by hcscherzer609** [gramps-project/gramps-web#492] [Ideas]
Did anybody think about integrating a wiki (or similar field) as an add-in (grampslet) to gramps or gramps we in order to view notes (text) combined with pictures on the main personal page?

**Comment by DavidMStraub** [comment]
What would be your use case? If you are tracking notes about ongoing research, in Gramps Web I am using the "description" field in tasks for note taking. Unfortunately, pictures inside notes are not currently supported by Gramps, see also #443.

**Comment by hcscherzer609** [comment]
Actually I have my data in a local desktop solution and am working alone.
For my family members I did set up a mediawiki installation.
This site I have to maintain manually and am looking for a solution with less effort to keep all information up to date.
Experience is, that nobody got used to work with the wiki and therefore the Gramps Web will be an alternative not only to show the famliy data but invite the members of the family to cooperate.
The possibility to present text combined with pictures will be - from my point of view - an outstanding feature.

**Comment by DavidMStraub** [comment]
This is worth elobarating further, but I'll convert it to a discussion; the pictures-inside-notes is tracked in #443 in the meantime.
